nvflare/lighter/poc_commands.py

```diff
@@ -0,0 +1,213 @@
+
+
+def start_poc(poc_workspace: str, white_list: list = []):
+    print(f"start_poc at {poc_workspace}, white_list={white_list}")
+    validate_poc_workspace(poc_workspace)
+    _run_poc(SC.CMD_START, poc_workspace, excluded=[SC.FLARE_OVERSEER], white_list=white_list)
+
+
+def stop_poc(poc_workspace: str, white_list: list = []):
+    print(f"stop_poc at {poc_workspace}")
+    validate_poc_workspace(poc_workspace)
+    _run_poc(SC.CMD_STOP, poc_workspace, excluded=[SC.FLARE_OVERSEER], white_list=white_list)
+
+
+def _build_commands(cmd_type: str, poc_workspace: str, excluded: list, white_list: list = []):
+    """
+    :param cmd_type: start/stop
+    :param poc_workspace:  poc workspace directory path
+    :param white_list: whitelist, service name. If empty, include every service
+    :param excluded: excluded service namae
+    :return:
+    """
+
+    service_commands = []
+    for root, dirs, files in os.walk(poc_workspace):
+        if root == poc_workspace:
+            for service_dir_name in dirs:
+                if service_dir_name not in excluded:
+                    if len(white_list) == 0 or service_dir_name in white_list:
+                        cmd = get_service_command(cmd_type, poc_workspace, service_dir_name)
+                        if cmd:
+                            service_commands.append((service_dir_name, cmd))
+    return sort_service_cmds(service_commands)
+
+
+def async_process(cmd_path):
+    import subprocess
+    import time
+
+    subprocess.Popen(cmd_path.split(" "))
+    time.sleep(3)
+
+
+def sync_process(cmd_path):
+    import subprocess
+
+    subprocess.run(cmd_path.split(" "))
+
+
+def _run_poc(cmd_type: str, poc_workspace: str, excluded: list, white_list=[]):
+    service_commands = _build_commands(cmd_type, poc_workspace, excluded, white_list)
+    for service_name, cmd_path in service_commands:
+        print(f"{cmd_type}: service: {service_name}, executing {cmd_path}")
+        if service_name == SC.FLARE_CONSOLE:
+            sync_process(cmd_path)
+        else:
+            async_process(cmd_path)
+
+
+def clean_poc(poc_workspace: str):
+    import shutil
+
+    if is_poc_ready(poc_workspace):
+        shutil.rmtree(poc_workspace, ignore_errors=True)
+    else:
+        print(f"{poc_workspace} is not valid poc directory")
+        exit(1)
+
+
+def def_poc_parser(sub_cmd, prog_name: str):
+    poc_parser = sub_cmd.add_parser("poc")
+    poc_parser.add_argument(
+        "-n", "--number_of_clients", type=int, nargs="?", default=2, help="number of sites or clients"
+    )
+    poc_parser.add_argument(
+        "-w", "--workspace", type=str, nargs="?", default=f"/tmp/{prog_name}/poc", help="poc workspace directory"
+    )
+    poc_parser.add_argument(
+        "-s",
+        "--service",
+        type=str,
+        nargs="?",
+        default="all",
+        help="service directory and sites, default to all = all services and sites, only used for"
+        "start/stop-poc commands",
+    )
```

Comment:
I personally prefer "packages" instead of "service".
What do you think @IsaacYangSLA @yanchengnv ?